setup.py

```diff
@@ -41,6 +41,7 @@
     install_requires=[
         'aspy.yaml',
         'cached-property',
+        'contextlib2',
```

Comment:
I actually run a youtube channel and my next episode in "porting to python 3" is about proper use of backports.  Haven't written up the slides yet or I'd link them but for backported modules you only want to conditionally install them on the versions that need them (otherwise we're installing contextlib2 in python3 where it's entirely unnecessary).

Here's the basic gist though: https://github.com/pre-commit/pre-commit/blob/v0.8.2/setup.py#L50-L52